frontend/src/app/rooms/[roomId]/page.js

```diff
@@ -56,11 +56,14 @@ function Page() {
       setScreen(questionScreen);
     });
 
-    socket.on("timerTick", ({timeLeft}) => {
+    socket.on("timerTick", ({timeLeft, correctAnswer}) => {
       setTimer(timeLeft);
+      setCorrectAnswer(correctAnswer);
```

Comment:
question, where is `setCorrectAnswer` being defined?

Reply:
if we want to do something like this, we need to define the function so at the top near where all the other usages of useState are:

```
const [correctAnswer, setCorrectAnswer] = useState(null);
```

Reply:
I made it so timerTick will have 
setCorrectAnswer(correctAnswer);

and it will setCorrectAnswer each time the timerTick is emitted specifically in startRound func in game-controller. 

I was having a tough time figuring out another way to do this but im sure there is one. I just know that this way produced the desired effect in my testing. 

Yes you are right i forgot to commit the const. for correctAnsweer, ive done so now